crates/stencil/src/function.rs

```diff
@@ -142,20 +139,19 @@ pub enum FunctionScript {
     Lazy(LazyFunctionScript),
 }
 
-/// Partially maps to FunctionCreationData in m-c/js/src/frontend/Stencil.h
+/// Partially maps to FunctionType in m-c/js/src/frontend/CompilationInfo.h
```

Comment:
We chatted about this in matrix -> this has probably been replaced with JSFunction.